src/pages/authorization/authorization.ts

```diff
@@ -0,0 +1,37 @@
+import Block from '../../template/block'
+import template from 'bundle-text:./authorization.hbs'
+import { checkInputValue, InputType } from '../../utils/mask'
+
+export class AuthorizationPage extends Block {
+    constructor() {
+        super()
+
+        this.setProps({
+            onSubmit: (e: any): void => {
+                e.preventDefault()
+                if (this.props.check()) {
+                    const data = {}
+                    Object.entries(this.refs).forEach((inputRef) => {
+                        data[inputRef[0] as string] = (inputRef[1] as HTMLInputElement).value
+                    })
```

Comment:
Если здесь копируется объект, то его можно скопировать более простым способом
```suggestion
                    const data = {...this.refs}
```

Reply:
> Если здесь копируется объект, то его можно скопировать более простым способом

Не совсем. Если, я все правильно поняла, то:
Я хочу получить объект вида -> { 'login': 'Vilena' }, а вашим способом получиться -> { 'login': input }.